src/exometer_report_influxdb.erl

```diff
@@ -136,6 +137,34 @@ exometer_report(Metric, DataPoint, _Extra, Value,
             Error
     end.
 
+exometer_report_bulk(_Found, _Extra, #state{connection = undefined} = State) ->
+    ?info("InfluxDB reporter isn't connected and will reconnect."),
+    {ok, State};
+exometer_report_bulk([{_Metric,[{_DataPoint, _Value}|_]}|_] = Found, _Extra, 
+                     #state{metrics = Metrics} = State) ->
+    {NState, Errors} = 
+        lists:foldl(
+          fun({Metric, DataPoints}, {StateAccIn, ErrorsAccIn}) -> 
+                  case maps:get(Metric, Metrics, not_found) of
+                      {MetricName, Tags} ->
+                          case maybe_send(Metric, MetricName, Tags, 
```

Comment:
@surik you're right. I missed the `batch_window_size` config. The original code supports the report in bulk.